sentences/en/light_HassLightSet.yaml

```
@@ -3,73 +3,41 @@ intents:
   HassLightSet:
     data:
       # brightness
+      # specific entity control
       - sentences:
-          - "[<numeric_value_set>] <name> brightness [to] <brightness>"
-          - "[<numeric_value_set>] [the] brightness [of] <name> [to] <brightness>"
-          - "[<numeric_value_set>] <name> [to] <brightness> [brightness]"
+          - "[<numeric_value_set>] ([of] <name>;[to] <light_value>)"
+          - "[<numeric_value_set>] ([of] <name>;[the] <brightness_synonyms>) [to] <light_value>"
+          - "[<numeric_value_set>] <name> [to] <light_value> <brightness_synonyms>"
         response: "brightness"
         requires_context:
           domain: light
 
+      # specific area/floor control
       - sentences:
-          - "<numeric_value_set> <name> to <brightness>"
-        requires_context:
-          domain: "light"
-        response: "brightness"
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness in <area> to <brightness>"
-          - "[<numeric_value_set>] [the] brightness of <area> to <brightness>"
-          - "[<numeric_value_set>] <area> brightness [to] <brightness>"
-          - "[<numeric_value_set>] <area> [to] <brightness> brightness"
-          - "[<numeric_value_set>] <area> [to] <brightness>"
-          - "[<numeric_value_set>] [(<all>|<the>)] <light> [<in>] <area> to <brightness> [brightness]"
-        response: "brightness"
-
-      - sentences:
-          - "<numeric_value_set> <area> to <brightness>"
-        response: "brightness"
-
-      - sentences:
-          - "[<numeric_value_set>] [the] brightness to <brightness>"
-          - "[<numeric_value_set>] [the] brightness (<in_here>;to <brightness>)"
+          - "[<numeric_value_set>] ([of] [(<all>|<the>)] <location>;[to] <light_value>)"
+          - "[<numeric_value_set>] ([of] [(<all>|<the>)] <location>;[the] <brightness_synonyms>) [to] <light_value>"
+          - "[<numeric_value_set>] [(<all>|<the>)] <location> ([to] <light_value>;[the] <brightness_synonyms>)"
         expansion_rules:
-          in_here: "[in] here"
+          location: "(<in_area_floor>|([<light>];<in_area_floor>))"
```

```suggestion
          location: "(<in_area_floor>|(<light>;<in_area_floor>))"
```
Not only are completely optional permutation members not recommended for the reasons I've mentioned, but having the other member listed alone as an alternative simply makes no sense.

Your version would yield `(<in_area_floor>|<in_area_floor>|<light> <in_area_floor>|<in_area_floor>|<in_area_floor> <light>)` whereas the suggestion would yield `(<in_area_floor>|<light> <in_area_floor>|<in_area_floor> <light>)`

---

Yep, this one was just a slip up on my part. Fixed now.